extras/scripts/fledge.service

```diff
@@ -68,15 +69,27 @@ get_pid() {
 
 fledge_start() {
     if [ "$IS_RHEL" = "" ]; then
-        sudo -u ${FLEDGE_USER} "${FLEDGE_ROOT}/bin/fledge" start > /dev/null
+        if [ -f "${FLEDGE_CERT}" ]; then
+            sudo -u ${FLEDGE_USER} "${FLEDGE_ROOT}/bin/fledge" -c "$FLEDGE_CERT}" start > /dev/null
+        else
+            sudo -u ${FLEDGE_USER} "${FLEDGE_ROOT}/bin/fledge" start > /dev/null
+	fi
+    elif [ -f "${FLEDGE_CERT}" ]; then
+        "${FLEDGE_ROOT}/bin/fledge" -c "${FLEDGE_CERT}" start > /dev/null
     else
         "${FLEDGE_ROOT}/bin/fledge" start > /dev/null
     fi
 }
  
 fledge_stop() {
     if [ "$IS_RHEL" = "" ]; then
-        sudo -u ${FLEDGE_USER} "${FLEDGE_ROOT}/bin/fledge" stop > /dev/null
+	if [ -f "${FLEDGE_CERT}" ]; then
+            sudo -u ${FLEDGE_USER} "${FLEDGE_ROOT}/bin/fledge" -c "${FLEDGE_CERT}" stop > /dev/null
+	else
+            sudo -u ${FLEDGE_USER} "${FLEDGE_ROOT}/bin/fledge" stop > /dev/null
```

Comment:
If the Authentication Method is configured to use a password, authentication will not succeed when executing systemctl stop.

Reply:
Hence having a sysctemctl user who we can mandate must use certificates.